index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Apple</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/867f1dd431.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <main>
+        <div class="mainContainer">
+            <ul class="navbar">
+                <li class="gradlink-style"><i class=" fab fa-apple"></i></li>
+                <li class="gradlink-style">Store</li>
+                <li class=" gradlink-style">Mac</li>
+                <li class="gradlink-style">iPod</li>
+                <li class=" gradlink-style">iPhone</li>
+                <li class="gradlink-style">iPad</li>
+                <li class=" gradlink-style">iTunes</li>
```

Comment:
These should be links. You can do that by adding an anchor tag

Reply:
anchor tag has been added.